vue-frontend/src/components/jobs/VirtuesView.vue

```diff
@@ -101,7 +101,7 @@
           class="tw-relative tw-flex-[100%] md:tw-flex-[33%]"
           v-for="virtue in virtues.slice(5, 8)"
           :key="virtue.id"
-          @touchstart="activeIndex = virtue.id"
+          @touchstart.passive="activeIndex = virtue.id"
```

Comment:
Have you checked it with real mobile devices?

Reply:
yes, its working